soc/xtensa/intel_adsp/cavs/include/intel_apl_adsp/adsp_shim.h

```diff
@@ -110,7 +110,7 @@ struct cavs_win {
 #define ADSP_DMWBA_ENABLE   BIT(0)
 #define ADSP_DMWBA_READONLY BIT(1)
 
-#define CAVS_CLKCTL_OSC_SOURCE_MASK  BIT_MASK(2)
-#define CAVS_CLKCTL_OSC_REQUEST_MASK (~BIT_MASK(28))
+#define ADSP_CLKCTL_OSC_SOURCE_MASK  BIT_MASK(2)
+#define ADSP_CLKCTL_OSC_REQUEST_MASK (~BIT_MASK(28))
```

Comment:
@abonislawski (https://github.com/zephyrproject-rtos/zephyr/pull/51919->ADSP_CLKCTL_OSC_REQUEST_MASK probably incorrect for ace)
Double-checked, and it's o.k. We can have a short call about this :)